### PR TITLE
Enable Raft by default in new clusters

### LINKED ISSUE #12572: Turn consistent-cluster-management ON by default in new clusters
We should turn consistent-cluster-management ON by default in new installations.

The feature is now generally available and is barely a feature - it fixes a number of gotchas in schema propagation code. The only downside (and why we call it a feature) is that now Scylla requires a majority of nodes to be around to do schema changes. This is acceptable in new deployments.

The fix is to change the default-installed scylla.yaml to turn the feature ON (the default is OFF). 

People using existing yaml files will continue using eventual schema propagation, unless consistent-cluster-management is explicitly requested.

### PR DESCRIPTION
**Pull request by kbr-scylla**
New clusters that use a fresh conf/scylla.yaml will have `consistent_cluster_management: true`, which will enable Raft, unless the user explicitly turns it off before booting the cluster.

People using existing yaml files will continue without Raft, unless consistent_cluster_management is explicitly requested during/after upgrade.

Also update the docs: cluster creation and node addition procedures.

Fixes #12572.

### CODE REVIEW

File: conf/scylla.yaml
Comment (kostja) [inline conf/scylla.yaml:562]: It actually can  - you can drop the raft state and never enable it back...

Comment (kostja) [reply conf/scylla.yaml:562]: I don't think we shoudl advertise this backdoor though...

**Review by kostja — APPROVED** [review]

**Comment by kbr-scylla** [comment]
@avikivity please review

**Review by avikivity — APPROVED** [review]

**Comment by kbr-scylla** [comment]
And please merge then :)

**Comment by kbr-scylla** [comment]
Ahh, maybe we should wait for review from @annastuchlik.
Anna could you please review the doc updates?

**Comment by avikivity** [comment]
> And please merge then :)

I'll let @tgrabiec have the honor.

**Review by annastuchlik — APPROVED** [review]
LGTM

**Comment by kbr-scylla** [comment]
Thanks. @tgrabiec please take a look

**Comment by kbr-scylla** [comment]
Queued on next-5.2

**Pull request merged**
